[Bug] [0.7.0] After creating the database, it cannot be deleted

### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar issues.


### What happened

After creating the database, it cannot be deleted

### What you expected to happen

logger content:

drop database db_tmp1 CASCADE 
Error message:
 org.apache.flink.table.api.TableException: Could not execute DROP DATABASE my_catalog.db_tmp1 CASCADE
	at org.apache.flink.table.api.internal.TableEnvironmentImpl.executeInternal(TableEnvironmentImpl.java:1083)
	at org.apache.flink.table.api.internal.TableEnvironmentImpl.executeSql(TableEnvironmentImpl.java:701)
	at com.dlink.executor.Executor.executeSql(Executor.java:247)
	at com.dlink.job.JobManager.executeSql(JobManager.java:421)
	at com.dlink.service.impl.StudioServiceImpl.executeFlinkSql(StudioServiceImpl.java:186)
	at com.dlink.service.impl.StudioServiceImpl.executeSql(StudioServiceImpl.java:173)
	at com.dlink.service.impl.StudioServiceImpl$$FastClassBySpringCGLIB$$e3eb787.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:793)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:89)
	at com.dlink.aop.UdfClassLoaderAspect.round(UdfClassLoaderAspect.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:634)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:624)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:72)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.dlink.service.impl.StudioServiceImpl$$EnhancerBySpringCGLIB$$af3a2303.executeSql(<generated>)
	at com.dlink.controller.StudioController.executeSql(StudioController.java:78)
	at com.dlink.controller.StudioController$$FastClassBySpringCGLIB$$e6483d87.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:793)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.adapter.AfterReturningAdviceInterceptor.invoke(AfterReturningAdviceInterceptor.java:57)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708)
	at com.dlink.controller.StudioController$$EnhancerBySpringCGLIB$$fe267e02.executeSql(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:150)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1071)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:964)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:696)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:779)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at com.alibaba.druid.support.http.WebStatFilter.doFilter(WebStatFilter.java:124)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:893)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1789)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.apache.flink.table.catalog.exceptions.CatalogException: 删除 database 信息失败：
	at com.dlink.flink.catalog.DlinkMysqlCatalog.dropDatabase(DlinkMysqlCatalog.java:448)
	at org.apache.flink.table.api.internal.TableEnvironmentImpl.executeInternal(TableEnvironmentImpl.java:1075)
	... 93 more
Caused by: java.sql.SQLSyntaxErrorException: Unknown column 'database_id' in 'where clause'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at com.dlink.flink.catalog.DlinkMysqlCatalog.dropDatabase(DlinkMysqlCatalog.java:443)
	... 94 more


### How to reproduce

1.create database tmp1;
2.drop database  tmp1 CASCADE;

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.13/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.13/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -407,7 +407,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             dStat.setInt(1, id);
             dStat.executeUpdate();
             dStat.close();
-            String deleteDbSql = "delete from metadata_database where database_id=?";
+            String deleteDbSql = "delete from metadata_database where id=?";
             dStat = conn.prepareStatement(deleteDbSql);
             dStat.setInt(1, id);
             dStat.executeUpdate();
@@ -438,7 +438,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
             String updateCommentSql =
-                    "update metadata_database set description=? where  database_id=?";
+                    "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.13/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.13/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -272,7 +272,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
                     pStat.setInt(1, id);
                     ResultSet prs = pStat.executeQuery();
                     while (prs.next()) {
-                        map.put(rs.getString("key"), rs.getString("value"));
+                        map.put(prs.getString("key"), prs.getString("value"));
                     }
                 } catch (SQLException e) {
                     sqlExceptionHappened = true;

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.13/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.13/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -437,8 +437,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
         try {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
-            String updateCommentSql =
-                    "update metadata_database set description=? where id=?";
+            String updateCommentSql = "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.14/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.14/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -277,7 +277,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
                     pStat.setInt(1, id);
                     ResultSet prs = pStat.executeQuery();
                     while (prs.next()) {
-                        map.put(rs.getString("key"), rs.getString("value"));
+                        map.put(prs.getString("key"), prs.getString("value"));
                     }
                 } catch (SQLException e) {
                     sqlExceptionHappened = true;

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.14/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.14/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -443,8 +443,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
         try {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
-            String updateCommentSql =
-                    "update metadata_database set description=? where id=?";
+            String updateCommentSql = "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.14/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.14/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -413,7 +413,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             dStat.setInt(1, id);
             dStat.executeUpdate();
             dStat.close();
-            String deleteDbSql = "delete from metadata_database where database_id=?";
+            String deleteDbSql = "delete from metadata_database where id=?";
             dStat = conn.prepareStatement(deleteDbSql);
             dStat.setInt(1, id);
             dStat.executeUpdate();
@@ -444,7 +444,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
             String updateCommentSql =
-                    "update metadata_database set description=? where  database_id=?";
+                    "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.15/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.15/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -411,7 +411,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             dStat.setInt(1, id);
             dStat.executeUpdate();
             dStat.close();
-            String deleteDbSql = "delete from metadata_database where database_id=?";
+            String deleteDbSql = "delete from metadata_database where id=?";
             dStat = conn.prepareStatement(deleteDbSql);
             dStat.setInt(1, id);
             dStat.executeUpdate();
@@ -442,7 +442,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
             String updateCommentSql =
-                    "update metadata_database set description=? where  database_id=?";
+                    "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.15/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.15/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -275,7 +275,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
                     pStat.setInt(1, id);
                     ResultSet prs = pStat.executeQuery();
                     while (prs.next()) {
-                        map.put(rs.getString("key"), rs.getString("value"));
+                        map.put(prs.getString("key"), prs.getString("value"));
                     }
                 } catch (SQLException e) {
                     sqlExceptionHappened = true;

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.15/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.15/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -441,8 +441,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
         try {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
-            String updateCommentSql =
-                    "update metadata_database set description=? where id=?";
+            String updateCommentSql = "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.16/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.16/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -277,7 +277,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
                     pStat.setInt(1, id);
                     ResultSet prs = pStat.executeQuery();
                     while (prs.next()) {
-                        map.put(rs.getString("key"), rs.getString("value"));
+                        map.put(prs.getString("key"), prs.getString("value"));
                     }
                 } catch (SQLException e) {
                     sqlExceptionHappened = true;

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.16/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.16/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -443,8 +443,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
         try {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
-            String updateCommentSql =
-                    "update metadata_database set description=? where id=?";
+            String updateCommentSql = "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

--- a/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.16/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
+++ b/dinky-catalog/dinky-catalog-mysql/dinky-catalog-mysql-1.16/src/main/java/org/dinky/flink/catalog/DinkyMysqlCatalog.java
@@ -413,7 +413,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             dStat.setInt(1, id);
             dStat.executeUpdate();
             dStat.close();
-            String deleteDbSql = "delete from metadata_database where database_id=?";
+            String deleteDbSql = "delete from metadata_database where id=?";
             dStat = conn.prepareStatement(deleteDbSql);
             dStat.setInt(1, id);
             dStat.executeUpdate();
@@ -444,7 +444,7 @@ public class DinkyMysqlCatalog extends AbstractCatalog {
             conn.setAutoCommit(false);
             // 1、名称不能改，类型不能改。只能改备注
             String updateCommentSql =
-                    "update metadata_database set description=? where  database_id=?";
+                    "update metadata_database set description=? where id=?";
             PreparedStatement uState = conn.prepareStatement(updateCommentSql);
             uState.setString(1, newDb.getComment());
             uState.setInt(2, id);

